native/helpers/challenges.js

```diff
@@ -453,4 +453,11 @@ export const challenges = [
   { title: 'Emoji Editor', link: '#' },
   { title: 'Sort Table', link: '#' },
   { title: 'Infinite Scroll', link: '#' },
+  {
+    title: 'Movie-Info-App',
+    link: 'Movie-Info-App',
+    difficulty: 'medium',
+    developer: 'sadanandpai',
+    tags: [],
+  },
```

Comment:
There should be your github username under developer.
Also add your details in the `helpers/contributors.js`.

Reply:
Also change the title to: `Movie-Info App (Online)`